src/ScrumBoardItBundle/Form/Type/Profile/FavoritesProfileType.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace ScrumBoardItBundle\Form\Type\Profile;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+
+/**
+ * Favorites Profile type.
+ *
+ * @author Brieuc Pouliquen <brieuc.pouliquen@canaltp.fr>
+ */
+class FavoritesProfileType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $favorites = $options['data'];
```

Comment:
That's pretty good you passed options via option. It would be more clear to name it favorites $options['favorites']